src/main/groovy/cruise/umple/UmpleGradlePlugin.groovy

```diff
@@ -29,12 +29,14 @@ class UmpleGradlePlugin implements Plugin<Project> {
 		project.task('generateSource') << {		
 			// command line arguments can be specified through gradle by -P (-P is for project properties)
 			// eg: "gradle compileUmpleFile -PUMPLE_FILE_PATH=test.ump -PLANGUAGE_TO_GENERATE=Java"	
+			m_umpleFilePath = "${project.projectDir}${File.separator}" 
+			m_generatedOutputPath = "${project.projectDir}${File.separator}"
 		
 			if(project.hasProperty(UMPLE_FILE_PATH))
 			{
-				m_umpleFilePath = project.getProperty(UMPLE_FILE_PATH)	
+				m_umpleFilePath += project.getProperty(UMPLE_FILE_PATH)	
```

Comment:
If you're talking about how the user specifies the paths, they should specify a path relative to the project directory. That's the way the plugin worked before and I wanted to keep it consistent.